src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.ListViewItemDetailsAccessibleObject.cs

```diff
@@ -11,21 +11,42 @@ public partial class ListViewItem
     {
         internal class ListViewItemDetailsAccessibleObject : ListViewItemBaseAccessibleObject
         {
+            private const int ImageIndex = 0;
             private readonly Dictionary<int, AccessibleObject> _listViewSubItemAccessibleObjects;
 
             public ListViewItemDetailsAccessibleObject(ListViewItem owningItem) : base(owningItem)
             {
                 _listViewSubItemAccessibleObjects = new Dictionary<int, AccessibleObject>();
             }
 
+            internal override int FirstSubItemIndex => HasImage ? 1 : 0;
+
+            private bool HasImage => _owningItem.ImageIndex != ImageList.Indexer.DefaultIndex;
+
+            private int LastChildIndex => HasImage ? _owningListView.Columns.Count : _owningListView.Columns.Count - 1;
+
+            /// <summary>
+            /// Convert subItemIndex to index. It depends on having <see cref="ListViewItemImageAccessibleObject"/>.
```

Comment:
We use double spaces on new line inside tags for documentation comments
```suggestion
            ///  Convert subItemIndex to index. It depends on having <see cref="ListViewItemImageAccessibleObject"/>.
```